Realm.xcodeproj/xcshareddata/xcschemes/Realm.xcscheme

```diff
@@ -52,6 +52,16 @@
                ReferencedContainer = "container:Realm.xcodeproj">
             </BuildableReference>
          </TestableReference>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "5D660FD71BE98C7C0021E04F"
+               BuildableName = "RealmSwift Tests.xctest"
+               BlueprintName = "RealmSwift Tests"
+               ReferencedContainer = "container:Realm.xcodeproj">
+            </BuildableReference>
+         </TestableReference>
```

Comment:
I don't think this should be here. It's intended that the RealmSwift tests are run only when the RealmSwift scheme is selected.